ketch-ios/ConsentConfig.swift

```diff
@@ -142,14 +140,6 @@ class ConsentHandler: NSObject, WKScriptMessageHandler {
         }
     }
 
-    private func payload<T: Decodable>(with payload: Any) -> T? {
```

Comment:
other data will still come with JSON, so I'd say we keep this function for such cases
only the CCPA and TCF strings will come as just strings.

Reply:
sure. we can return this code from history when necessary. just trying to keep the repo without dead code

Reply:
@anton-lyfar it is not dead, we actually need this JSON example - otherwise developers will not know how we recommend to encode it